pkg/kademlia/testrouting/viz.go

```diff
@@ -0,0 +1,66 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package testrouting
+
+import (
+	"encoding/hex"
+	"fmt"
+	"io"
+)
+
+// Graph writes a DOT format visual graph description of the routing table to w
+func (t *Table) Graph(w io.Writer) error {
```

Comment:
Serialize into `bytes.Buffer` then you don't have to worry about error handling and can use things like:

```
fmt.Fprintln(&buf, "digraph {\n  node [shape=box];\n")
defer fmt.Fprintln(&buf, "}")

...
```

Reply:
alternatively a local func to hide error handling https://github.com/loov/layout/blob/master/format/dot/write.go#L10

Reply:
kk